lib/ui/src/components/layout/desktop.js

```diff
@@ -0,0 +1,136 @@
+import { localStorage } from 'global';
```

Comment:
I'm thinking of overhauling still once more...

Currently enabling or disabling / moving panels has the undesired consequence that the iframe might reload.

IF...

I just place resize detectors inside the panels, and keep track of the desired sizes that way...
I can just absolutely position the true panels.

Hope this makes sense.

This would result in the panels never moving from the dom/reparenting.